example/webapp/src/index.tsx

```diff
@@ -4,7 +4,7 @@
 import * as React from "react";
```

Comment:
You'll need to update the `.gitpod.yml` so that 

```
tasks:
- before: cd webapp
  init: yarn
  command: nginx -c $PWD/nginx.conf; npx webpack-dev-server
```
is done in the correct directory.

that also should prevent you from having to change the import path on eosjs